### ISSUE TITLE
how to use .pth weight file and

### ISSUE THREAD

**Issue by liwanjie1020** [eriklindernoren/PyTorch-YOLOv3#185]
After training data set of  myself, in the "checkpoints" file generated ". Pyh "weight file, then what should I test my test set (in other words: what should I input command)

**Comment by sourav164** [comment]
Have you figured it out? I am having the same issue?

**Comment by sourav164** [comment]
@liwanjie1020  Have you figured it out? I am having the same issue?

**Comment by kaikaizhu** [comment]
@sourav164 
Have you figured it out? I am having the same issue too.

**Comment by sourav164** [comment]
@kaikaizhu  nope. This model was running for me but it was not detecting anything at all. Thus, I am using the tensorflow version of it.

**Comment by NiubilityYang** [comment]
also facing the same issue

**Comment by NiubilityYang** [comment]
it is written in detect.py, the command should be like this: python detect.py --image_folder data/images  --checkpoint_model checkpoints/yolov3_ckpt_1.pth

**Comment by HaochenW** [comment]
When you train your own dataset, then you want to test it by checkpoint .pth file, it could be better to try: 
python detect.py --image_folder data/VOC2007_test/ --weights_path checkpoints/yolov3_ckpt_73.pth --model_def config/yolov3-custom.cfg

**Comment by nanhui69** [comment]
@NiubilityYang  do you run the detect.py，when i run the detect.py,the dection is None,what's the problem?

**Comment by priteshgohil** [comment]
Same problem, I tried to change the .pth -> .weights

But didn't help. I'm getting None result

**Comment by priteshgohil** [comment]
Update: This worked for me
`python detect_custom.py --model_def config/yolov3-custom.cfg --weights_path checkpoints/yolov3_ckpt_100.pth --image_folder data/samples/ --class_path data/custom/classes.names `
